src/metabase/driver/common.clj

```diff
@@ -126,6 +127,22 @@
    :ssl                default-ssl-details
    :user               default-user-details})
 
+(def cloud-ip-address-info
+  "Map of the `cloud-ip-address-info` info field. The getter is invoked and converted to a `:placeholder` value prior
+  to being returned to the client, in [[metabase.driver.util/connection-props-server->client]]."
+  {:name   "cloud-ip-address-info"
+   :type   :info
+   :getter (fn []
+             (when-let [ips (public-settings/cloud-gateway-ips)]
+               (str (deferred-tru "If your database is behind a firewall, you may need to allow connections from our Metabase Cloud IP addresses:")
+                    "\n"
+                    (clojure.string/join " - " (public-settings/cloud-gateway-ips)))))})
```

Comment:
https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#require-namespaces-you-use

It's always good to explicitly require the namespace in `:require`. Otherwise, this code won't work unless `clojure.string` has already been loaded somewhere else before we get to this point

Reply:
Yes, this is still a WIP :) haven't polished it up